msprime/cli.py

```diff
@@ -628,6 +681,7 @@ def get_mspms_parser(error_handler=None):
         "--gene-conversion", "-c", type=float, nargs=2, default=(0, 1),
         metavar=("gc_recomb_ratio", "track_length"),
         help=mscompat_gene_conversion_help)
+    group.add_argument("--hotspots", "-v", type=float, nargs="+", default=None)
```

Comment:
Needs some help text. Can say something like "Recombination hotspots defined according to msHOT format. This is blah blah``